docs/en/modules/ucg.md

```diff
@@ -48,13 +48,13 @@ The NodeMCU firmware supports a subset of these:
 - SSD1351
 - ST7735
 
-This integration is based on [v1.5.2](https://github.com/olikraus/Ucglib_Arduino/releases/tag/v1.5.2).
+This integration is based on version [1.5.2](https://github.com/olikraus/Ucglib_Arduino/releases/tag/1.5.2).
 
 ## Overview
 
 ### SPI Connection
 
-The HSPI module is used ([more information](http://d.av.id.au/blog/esp8266-hardware-spi-hspi-general-info-and-pinout/)), so certain pins are fixed:
+The HSPI module is used ([more information](spi)), so certain pins are fixed:
```

Comment:
This looks wrong

Reply:
In `spi` module there is documentation about the HSPI pinout, but on MetalPhreak's blog posts there is more info so I understand. Changed it to Web Archive snapshot - it's easier to link to external resources than internal docs because of the problems with MkDocs - those are working on most of MkDocs versions